saleor/graphql/checkout/mutations/utils.py

```diff
@@ -273,24 +275,100 @@ def get_checkout(
     return checkout
 
 
-def group_quantity_and_custom_prices_by_variants(
-    lines: List[Dict[str, Any]]
+def group_lines_input_on_add(
+    lines: List[Dict[str, Any]], existing_lines_info=None
 ) -> List[CheckoutLineData]:
-    variant_checkout_line_data_map: Dict[str, CheckoutLineData] = defaultdict(
-        CheckoutLineData
-    )
+    """Return list od CheckoutLineData objects.
+
+    This function is used in CheckoutLinesAdd and CheckoutCreate mutation.
```

Comment:
Not sure if we should explicitly said in the docstring, where the function is used. If I use it in different place, then to keep a consistency I will have to also update the description.
If I change the name of mutation (or something), the description of the function will be incorrect.